src/dict/mediawiki.cc

```diff
@@ -154,6 +157,7 @@ MediaWikiWordSearchRequest::MediaWikiWordSearchRequest( wstring const & str,
   GlobalBroadcaster::instance()->addWhitelist( reqUrl.host() );
 
   Utils::Url::addQueryItem( reqUrl, "apprefix", QString::fromStdU32String( str ).replace( '+', "%2B" ) );
+  Utils::Url::addQueryItem( reqUrl, "lang", lang );
```

Comment:
check lang.isEmpty

Reply:
The empty lang shouldn't be an issue, b/c the final request url will be some thing like `https://...?...&lang=&...` which won't impact the requested page.